adapter/internal/api/apis_impl.go

```diff
@@ -272,9 +280,9 @@ func ApplyAPIProjectFromAPIM(payload []byte, vhostToEnvsMap map[string][]string)
 		// first update the API for vhost
 		deployedRevision, err := updateAPI(vhost, apiInfo, apiProject, allEnvironments)
 		if err != nil {
-			return deployedRevisionList, err
+			return deployedRevisionList, fmt.Errorf("%v:%v with UUID \"%v\"", apiInfo.Name, apiInfo.Version, apiInfo.ID)
 		}
-        if deployedRevision != nil {
+		if deployedRevision != nil {
```

Comment:
shall we fix the formatting issue

Reply:
This was a fix for a formatting issue that already existed.